src/main/java/ch/heigvd/pro/b04/answers/Answer.java

```diff
@@ -1,7 +1,14 @@
 package ch.heigvd.pro.b04.answers;
 
+import ch.heigvd.pro.b04.participants.Participant;
```

Comment:
Useless import

Reply:
If there's only this.... it's a detail seriously